tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -758,7 +758,7 @@ def test_related_institutes_from__510_a_w_0():
     assert expected == result['related_institutes']
 
 
-def test_related_institutes_from__double_510_a_w_0():
+def test_related_institutes_from__double_510_w_0():
```

Comment:
Why are you changing the test names? The test names denote the snippet that you're using. If the snippet doesn't change (and there's usually no reason to change the snippet), the test name doesn't change either!